spinners.json

```diff
@@ -835,5 +835,40 @@
 			"p",
 			"b"
 		]
-	}
+	},
+  "weather": {
+    "interval": 100,
+    "frames": [
```

Comment:
The indentation is still incorrect. It should be the same as the rest of the file (tab).